apps/win32/README.md

```diff
@@ -1,15 +1,10 @@
-`FluentUI Tester` is the test app that we use to test our FluentUI components during development.
+# Running the FluentUI Tester on Win32
 
-## Prerequisites
+`FluentUI Tester` is the test app that we use to test our FluentUI components during development. It uses the [react-native-test app](https://github.com/microsoft/react-native-test-app) under the covers, and loads the fluent-tester bundle.
```

Comment:
`react-native-test-app` doesn't support Win32 yet :smile:

We do have support for Windows and there's a task for it: #447

Reply:
Copy paste error, good catch.

Reply:
As for building the Windows FluentUI tester on react-native-test-app, thank you for creating that task.